src/PIL/Image.py

```diff
@@ -1318,6 +1319,27 @@ def getexif(self):
 
         return self._exif
 
+    def getxmp(self):
+        """
+        Returns an object containing the xmp tags for a given image.
+        :returns: XMP tags in an object.
+        """
+
+        if self._xmp is None:
+            self._xmp = {}
+
+        for segment, content in self.applist:
+            if segment == "APP1":
+                marker, xmp_tags = content.rsplit(b"\x00", 1)
+                if marker == b"http://ns.adobe.com/xap/1.0/":
+                    root = xml.etree.ElementTree.fromstring(xmp_tags)
+                    for element in root.findall(".//"):
+                        xmp_atribs = []
+                        for child, value in element.attrib.items():
+                            xmp_atribs.append({child.split("}")[1]: value})
```

Comment:
This seems odd to me. Why have a list of single item dictionaries, when it could just be one dictionary? See https://github.com/UrielMaD/Pillow/pull/10 for an implementation.

Reply:
I could just leave the dictionary containing just the items of "Description" where are the most of the xmp tags, I just thought that it could be nice to have more info. I'll modify it right away!

Reply:
I'm not following. I don't want to lose any information. In your current version, it returns
`{'RDF': [], 'Description': [{'Version': '10.4'}, {'ProcessVersion': '10.0'}, {'ConvertToGrayscale': 'False'}, {'CameraProfile': 'Adobe Standard'}, {'LookTable': 'E1095149FDB39D7A057BAB208837E2E1'}], 'History': [], 'Seq': [], 'li': [], 'DerivedFrom': [{'documentID': '8367D410E636EA95B7DE7EBA1C43A412'}, {'originalDocumentID': '8367D410E636EA95B7DE7EBA1C43A412'}], 'ToneCurve': [], 'ToneCurveRed': [], 'ToneCurveGreen': [], 'ToneCurveBlue': [], 'ToneCurvePV2012': [], 'ToneCurvePV2012Red': [], 'ToneCurvePV2012Green': [], 'ToneCurvePV2012Blue': [], 'RetouchAreas': [], 'Masks': [], 'RetouchInfo': [], 'Look': [], 'Group': [], 'Alt': [], 'Parameters': []}`

In what I'm suggesting, it returns
`{'RDF': {}, 'Description': {'Version': '10.4', 'ProcessVersion': '10.0', 'ConvertToGrayscale': 'False', 'CameraProfile': 'Adobe Standard', 'LookTable': 'E1095149FDB39D7A057BAB208837E2E1'}, 'History': {}, 'Seq': {}, 'li': {}, 'DerivedFrom': {'documentID': '8367D410E636EA95B7DE7EBA1C43A412', 'originalDocumentID': '8367D410E636EA95B7DE7EBA1C43A412'}, 'ToneCurve': {}, 'ToneCurveRed': {}, 'ToneCurveGreen': {}, 'ToneCurveBlue': {}, 'ToneCurvePV2012': {}, 'ToneCurvePV2012Red': {}, 'ToneCurvePV2012Green': {}, 'ToneCurvePV2012Blue': {}, 'RetouchAreas': {}, 'Masks': {}, 'RetouchInfo': {}, 'Look': {}, 'Group': {}, 'Alt': {}, 'Parameters': {}}`

Reply:
I think the changes you suggest are better, I merged them into my PR, so I think now it's better